### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SeriousMatt**

### CODE REVIEW

**Comment by SeriousMatt** [comment]
Updated the Readme.md after @hlbraddock noticed some problems with getting his jobs added.

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ How to Add or Update a Job
 * Test your work
   * Push your feature branch to github
   * Request AWSInfrastructure@cru.org to turn on Jenkins Lab, if jenkins-lab.cru.org gives you a 503
+  * Merge your jenkins-job branch to jenkins-lab branch
```
Comment (mattdrees) [inline README.md]: So, the cru-ansible setup on jenkins-lab is smart enough to run the jenkins-lab branch instead of master?

Comment (SeriousMatt) [reply README.md]: @mattdrees that's the idea. Devs do not have access to the configure page.

Comment (mattdrees) [reply README.md]: so, I see that https://jenkins-lab.cru.org/job/create-jenkins-jobs/configure has ` --extra-vars="jenkins_jobs_version=jenkins-lab"`, but that is not in this repo's simple-jobs.yaml. Did you manually add that? And if so, won't it get overwritten if we run the job?

Comment (mattdrees) [reply README.md]: By the way, I'm not arguing against your proposed workflow; it is better for sure.
It's just that I want to make sure it's not broken.

Comment (SeriousMatt) [reply README.md]: Thanks @mattdrees. It will need to be added there as well. I'll make the change. Appreciate the feedback! :)

Comment (SeriousMatt) [reply README.md]: and yes it was added manually. I was not sure if it was configured with ansible or not. Good to see it's configured in this repo.

**Comment by mattdrees** [comment]
So, now the problem is the jenkins-lab branch will be used by jenkins-prod.

Is there a way you can add a bash if statement in there that figures out whether it is running on jenkins-lab or jenkins-prod, and if it's jenkins-lab, then use `--extra-vars` arg, but not otherwise?

**Comment by SeriousMatt** [comment]
@mattdrees that's possible. I'm circling back on how this should be handled. Not really liking breaking the yml syntax by adding a bash. Is it better to leave this one change unmerged in the jenkins-lab branch with the --extra-vars set to jenkins-lab (its a chicken or the egg kind of problem)? This assumes that the jenkins-lab branch really is the jenkins-lab server and the master branch is jenkins-prod. Maybe we could use ansible to configure jenkins to use the jenkin-lab branch for jenkins-jobs and leave the jenkins-job simple-jobs.yml alone. Just some thoughts.

**Comment by mattdrees** [comment]
> Not really liking breaking the yml syntax by adding a bash

Not quite following. It's a 'shell' step; it's already bash.

**Pull request closed without merging**